Makefile

```diff
@@ -887,20 +887,11 @@ bump-eve-api:
 
 .PHONY: proto-api-%
 
-check-patch-%:
-	@if ! echo $* | grep -Eq '^[0-9]+\.[0-9]+$$'; then echo "ERROR: must be on a release branch X.Y"; exit 1; fi
-	@if ! echo $(EVE_TREE_TAG) | grep -Eq '^$*.[0-9]+-'; then echo "ERROR: can't find previous release's tag X.Y.Z"; exit 1; fi
+rc-release:
+	@bash tools/rc-release.sh
 
-patch-%: check-patch-%
```

Comment:
Please, update the "help" target inside the Makefile accordingly, i.e., remove the patch and add the description for the new targets: rc-relase and lts-release